### DISCUSSION TITLE
When is the next stable version of discord.py going to be released?

### DISCUSSION THREAD

**Discussion by 97I8TOE47K** [Rapptz/discord.py#8084] [General]
Still waiting for a stable release with built-in slash commands and modals! 🥲

**Comment by LeoCx1000** [comment]
There is no ETA, unfortunately. But you can still [install v2.0 beta from source](https://github.com/rapptz/discord.py#:~:text=To%20install%20the%20development%20version%2C%20do%20the%20following%3A) to get those features.

**Comment by lol219** [comment]
Just wait for a while, or you can use the github version, but you can get some bugs since it's alpha.
Or if you want stability, you can stick with the pypi version until v2.0 stable get released.
